<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>791</width>
      <height>511</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="OpenGLWidget" name="openGLWidget"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetNoConstraint</enum>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Lab1:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="primitivesBox">
         <property name="maximumSize">
          <size>
           <width>232</width>
           <height>16777215</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>GL_POINTS</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GL_LINES</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GL_LINE_STRIP</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GL_LINE_LOOP</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GL_TRIANGLES</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GL_TRIANGLE_STRIP</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GL_TRIANGLE_FAN</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GL_QUADS</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GL_QUAD_STRIP</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GL_POLYGON</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="maximumSize">
          <size>
           <width>232</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Lab2:
Alpha parameters:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="alphaFuncBox">
         <property name="maximumSize">
          <size>
           <width>232</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="currentText">
          <string>GL_ALWAYS</string>
         </property>
         <item>
          <property name="text">
           <string>GL_NEVER</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GL_LESS</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GL_EQUAL</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GL_LEQUAL</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GL_GREATER</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GL_NOTEQUAL</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GL_GEQUAL</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GL_ALWAYS</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="alphaRefSlider">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>232</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="maximumSize">
          <size>
           <width>232</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Blend parameters:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="blendBox_1">
         <property name="maximumSize">
          <size>
           <width>232</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="currentText">
          <string>GL_ONE</string>
         </property>
         <item>
          <property name="text">
           <string>GL_ZERO</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GL_ONE</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GL_DST_COLOR</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GL_ONE_MINUS_DST_COLOR</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GL_SRC_ALPHA</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GL_ONE_MINUS_SRC_ALPHA</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GL_DST_ALPHA</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GL_ONE_MINUS_DST_ALPHA</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GL_SRC_ALPHA_SATURATE</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="blendBox_2">
         <property name="maximumSize">
          <size>
           <width>232</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="currentText">
          <string>GL_ONE_MINUS_SRC_ALPHA</string>
         </property>
         <item>
          <property name="text">
           <string>GL_ZERO</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GL_ONE</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GL_SRC_COLOR</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GL_ONE_MINUS_SRC_COLOR</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GL_SRC_ALPHA</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GL_ONE_MINUS_SRC_ALPHA</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GL_DST_ALPHA</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GL_ONE_MINUS_DST_ALPHA</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>GL_SRC_ALPHA_SATURATE</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="maximumSize">
          <size>
           <width>232</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Scissors parameters:
x</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="xSlider">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>232</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>9</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>y</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="ySlider">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>232</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>52</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Width</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="wSlider">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>232</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>59</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Height</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="hSlider">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>232</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>100</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>813</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OpenGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>openglwidget.h</header>
   <slots>
    <slot>setPrimitive(QString)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
